language/move-lang/tests/move_check/expansion/standalone_mname_with_type_args.exp

```diff
@@ -1,14 +1,8 @@
 error: 
 
-   ┌── tests/move_check/expansion/standalone_mname_with_type_args.move:3:27 ───
+   ┌── tests/move_check/expansion/standalone_mname_with_type_args.move:3:21 ───
    │
  3 │         let _ = 0 + M<u64>;
-   │                           ^ Unexpected ';'
-   ·
- 3 │         let _ = 0 + M<u64>;
-   │                           - Expected either a brace-enclosed list of field expressions or a parenthesized list of arguments for a function call
-   ·
- 3 │         let _ = 0 + M<u64>;
-   │                      - Perhaps you need a blank space before this '<' operator?
+   │                     ^ Expected either a brace-enclosed list of field expressions or a parenthesized list of arguments for a function call
```

Comment:
That message only works if the location is right past the closing "\>". Alternatively, the span used in this message could include the name and type argument list and the message rephrased to something like "Expected name to be followed by either..." @tnowacki, do you have any other ideas for this message? Is there any way we could defer the error until we know whether the name refers to a function or a struct?

Reply:
See whether you like it now better.

Reply:
Much better!